rmgpy/rmg/model.py

```diff
@@ -1142,7 +1142,7 @@ def removeEmptyPdepNetworks(self):
                     del(self.networkDict[source])
                 self.networkList.remove(network)
                     
-    def prune(self, reactionSystems, toleranceKeepInEdge, maximumEdgeSpecies, minSpeciesExistIterationsForPrune):
+    def prune(self, reactionSystems, toleranceKeepInEdge, toleranceMoveToCore, maximumEdgeSpecies, minSpeciesExistIterationsForPrune):
```

Comment:
Is prune() also being called with `toleranceMoveToCore`?

Reply:
This should be fixed now